common/djangoapps/student/views.py

```diff
@@ -178,10 +178,15 @@ def index(request, extra_context=None, user=AnonymousUser()):
     courses = get_courses(user)
 
     if configuration_helpers.get_value(
-            "ENABLE_COURSE_SORTING_BY_START_DATE",
-            settings.FEATURES["ENABLE_COURSE_SORTING_BY_START_DATE"],
+        "ENABLE_COURSE_SORTING_BY_START_DATE",
+        settings.FEATURES["ENABLE_COURSE_SORTING_BY_START_DATE"],
```

Comment:
What would happen if `ENABLE_COURSE_SORTING_BY_START_DATE` key is not found in the settings.FEATURES?

Reply:
This key has "True"  as default value in "lms/envs/common.py" (https://github.com/raccoongang/edx-platform/pull/1177/files#diff-82bda515879c96ab0641bfcbe58631c3R257).